README.rst

```diff
@@ -210,6 +211,8 @@ Placeholder     Definition
 ``{commit}``    The hash of the commit the build ran against
 ``{number}``    The run number of the workflow run (GitHub) or the build number
                 (Travis and Appveyor)
+``{status}``    The success status of the build; the exact strings used depend
```

Comment:
could we provide `{status_harmonized}` or alike which would bring them to some common dictionary?  how different are they?

Reply:
* The GitHub Actions conclusions are `action_required`, `cancelled`, `failure`, `neutral`, `success`, `skipped`, `stale`, and `timed_out`.
* I can't find documentation on the Travis states, but they include `passed`, `errored`, and `failed`.
* I can't find documentation on the Appveyor statuses, but they include `success`, `failed`, and `cancelled`.

Reply:
> * The GitHub Actions conclusions are `action_required`, `cancelled`, `failure`, `neutral`, `success`, `skipped`, `stale`, and `timed_out`.
> * I can't find documentation on the Travis states, but they include `passed`, `errored`, and `failed`.
> * I can't find documentation on the Appveyor statuses, but they include `success`, `failed`, and `cancelled`.

Well, the main purpose is to be able to tell good from not good and the one which did not run/complete at all. 
So appveyor seems to come closest to such coarse level, but would be sad to loose the failed-vs-errored distinction.  So what about something like

```python
{ # harmonized: {original}
"success": {"success", "passed"},
"failed": {"failure", "failed"},
"errored": { "errored", "timed_out"},
"incomplete": {"action_required", "cancelled", "skipped", "stale"},
"other": { ... anything which is not above }
}
```
or may be even without `other` so if we run into some -- we have to categorize it into 4.  WDYT?

Reply:
`{common_status}` placeholder added.